V2/tezex/src/contexts/network.tsx

```diff
@@ -0,0 +1,49 @@
+import { createContext } from "react";
+import { NetworkType } from "@airgap/beacon-sdk";
+
+export interface Addresses {
+	sirs: { address: string };
+	tzbtc: { address: string; dex: { sirius: string } };
+}
+export interface NetworkInfo {
+	network: NetworkType;
+	tezosServer: string;
+	tezosServerForTaquito: string;
+	apiKey: string;
+	addresses: Addresses;
+}
+
+export type NetworkMap = {
+	[network: string]: NetworkInfo;
+};
+export interface INetwork {
+	selectedNetwork: NetworkType;
+	networks: NetworkMap;
+}
+
+export const mainnet = {
+	network: NetworkType.MAINNET,
+	tezosServer: "https://mainnet.api.tez.ie",
+	tezosServerForTaquito: "",
+	apiKey: "ab682065-864a-4f11-bc77-0ef4e9493fa1",
```

Comment:
this should be in a secrets file

Reply:
This should never have been committed(apiKey), its not even being used , it was a mistake , should the server also be in a secrets or just a config file?